packages/app/src/specs/banners/LoginBanner.cy.tsx

```diff
@@ -0,0 +1,9 @@
+import LoginBanner from './LoginBanner.vue'
+
+describe('<LoginBanner />', () => {
+  it('should render expected content', () => {
+    cy.mount({ render: () => <LoginBanner modelValue={true} /> })
+
+    cy.percySnapshot()
```

Comment:
Similar to an earlier comment, I'd like this test to fail if the content were missing. Percy will catch this too but is a lot slower/sometimes gets ignored.

Or maybe we have an e2e test or higher level component test that checks the contents, and I haven't see it yet.